ui/src/containers/AppVersionsTableContainer.js

```diff
@@ -3,22 +3,55 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Checkbox, TextInput } from '@patternfly/react-core';
 import moment from 'moment';
-import { getApps, appDetailsSort, updateDisabledAppVersion, updateVersionCustomMessage } from '../actions/actions-ui';
+import { getApps, appDetailsSort } from '../actions/actions-ui';
 import AppsTable from '../components/AppsTable';
 import './TableContainer.css';
 import config from '../config/config';
 
 export class AppVersionsTableContainer extends React.Component {
+  shouldComponentUpdate = () => {
+    return this.state.shouldAppUpdate;
+  }
+  componentWillMount = () => {
+    this.setState({
```

Comment:
2 points;
- if tracking local state in the component I think we should use React hooks to be consistent with other scheduled refactoring that will see us use React hooks to manage local state in our components
- if using `setState`, I think we should be using a constructor to declare the initial state (react docs seem unclear if this is necessary to manipulate state in the class, but it would appear to be good practice in any event)